src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let result = 0;
+
+  const add = (value) => {
+    if (!value && value !== 0) {
```

Comment:
double check?
```suggestion
    if (value === undefined) {
```

Reply:
My mistake. I was thinking about checking the passed parameter, which should be checked after this. Something like this: